### ISSUE TITLE
Crash on loading The Lurking Horror blorb after Arthur blorb

### ISSUE THREAD

**Issue by ptomato** [wmvanvliet/Chimara#21]
Opened 2011-04-20 by **Ezekiel000**

I found that every time I load the zlb (zblorb) or Arthur and then open The Lurking Horror zlb without exiting Chimara. Chimara crashes with a bug-buddy report generated.

**Comment by ptomato** [comment]
```
System: Linux 2.6.35-28-generic #50-Ubuntu SMP Fri Mar 18 18:42:20 UTC 2011 x86_64
X Vendor: The X.Org Foundation
X Vendor Release: 10900000
Selinux: No
Accessibility: Disabled
GTK+ Theme: Clearlooks
Icon Theme: gnome-brave
GTK+ Modules: gnomesegvhandler, canberra-gtk-module

Memory status: size: 413433856 vsize: 413433856 resident: 19718144 share: 12681216 rss: 19718144 rss_rlim: 18446744073709551615
CPU usage: start_time: 1303328806 rtime: 88 utime: 78 stime: 10 cutime:0 cstime: 0 timeout: 0 it_real_value: 0 frequency: 100



---- Critical and fatal warnings logged during execution ----

** Chimara **: glk_window_open: Tried to open a new root window, but there is already a root window
** Chimara **: glk_window_open: Tried to open a new root window, but there is already a root window


----------- .xsession-errors ---------------------
** (gnome-session:2154): DEBUG: GsmDBusClient: obj_path=/org/freedesktop/DBus interface=org.freedesktop.DBus method=NameOwnerChanged
** (gnome-session:2154): DEBUG: GsmDBusClient: obj_path=/org/freedesktop/DBus interface=org.freedesktop.DBus method=NameOwnerChanged
** (gnome-session:2154): DEBUG: GsmDBusClient: obj_path=/org/freedesktop/DBus interface=org.freedesktop.DBus method=NameOwnerChanged
** (gnome-session:2154): DEBUG: GsmDBusClient: obj_path=/org/freedesktop/DBus interface=org.freedesktop.DBus method=NameOwnerChanged
** (gnome-session:2154): DEBUG: GsmDBusClient: obj_path=/org/freedesktop/DBus interface=org.freedesktop.DBus method=NameOwnerChanged
** (gnome-session:2154): DEBUG: GsmDBusClient: obj_path=/org/freedesktop/DBus interface=org.freedesktop.DBus method=NameOwnerChanged
** (gnome-session:2154): DEBUG: GsmDBusClient: obj_path=/org/freedesktop/DBus interface=org.freedesktop.DBus method=NameOwnerChanged
** (gnome-session:2154): DEBUG: GsmDBusClient: obj_path=/org/freedesktop/DBus interface=org.freedesktop.DBus method=NameOwnerChanged
** (gnome-session:2154): DEBUG: GsmDBusClient: obj_path=/org/freedesktop/DBus interface=org.freedesktop.DBus method=NameOwnerChanged
Could not attach to process.  If your uid matches the uid of the target
process, check the setting of /proc/sys/kernel/yama/ptrace_scope, or try
again as the root user.  For more details, see /etc/sysctl.d/10-ptrace.conf
ptrace: Operation not permitted.
/home/ezekiel/Games/Frotz/Save/19632: No such file or directory.
No stack.
--------------------------------------------------
```

**Comment by ptomato** [comment]
@wmvanvliet commented 2011-06-18:

> The .z files work for me. Can you attach the zblorbs in question?

**Ezekiel000** commented 2011-11-07:

> Too big to attach but here is a link to the two zblorbs:
> ​http://www.mediafire.com/?cagtdt6us28il9p

@ptomato closed 2012-09-24 from commit [61180da]
